webauthn-rs-proto/src/options.rs

```diff
@@ -179,6 +179,56 @@ impl ToString for AuthenticatorTransport {
     }
 }
 
+/// The type of attestation on the credential
+///
+/// <https://www.iana.org/assignments/webauthn/webauthn.xhtml>
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Hash)]
+pub enum AttestationFormat {
+    /// Packed attestation
+    #[serde(rename = "packed", alias = "Packed")]
+    Packed,
+    /// TPM attestation (like Micrsoft)
+    #[serde(rename = "tpm", alias = "Tpm", alias = "TPM")]
+    Tpm,
+    /// Android hardware attestation
+    #[serde(rename = "android-key", alias = "AndroidKey")]
+    AndroidKey,
+    /// Older Android Safety Net
+    #[serde(
+        rename = "android-safetynet",
+        alias = "AndroidSafetyNet",
+        alias = "AndroidSafteynet"
+    )]
+    AndroidSafetynet,
```

Comment:
This changed from `AndroidSafetyNet` to `AndroidSafetynet` in the move.

It looks like `SafetyNet` is the usual capitalisation:

* https://developer.android.com/privacy-and-security/safetynet
* https://developer.android.com/privacy-and-security/safetynet/attestation
* https://developer.android.com/privacy-and-security/safetynet/deprecation-timeline

Reply:
Yeah, I renamed it to serilise to android-safetynet, but given I added the manual rename we don't need it.